load-balancing/elb-v2/common_functions.sh

```diff
@@ -374,15 +380,23 @@ wait_for_state() {
         return 1
     fi
 
-    msg "Checking $WAITER_ATTEMPTS times, every $waiter_interval seconds, for instance $instance_id to be in state $state_name"
+    # Check if a custom waiter_attempts was passed into the function
+    # and override the attemps if true
+    if [ -n ${var+5} ]; then
+        local waiter_attempts=$5
```

Comment:
It is supposed to be $5. I want to set it as the fifth argument if it doesn't already exist.